### PR TITLE
Chore/refactor for tarball contents renaming

### PR DESCRIPTION
**Pull request by lozette**

### CODE REVIEW

File: ds-caselaw-ingester/lambda_function.py [hunk 1]
```diff
@@ -38,23 +50,21 @@ def extract_lambda_versions(versions: List[Dict[str, str]]) -> List[Tuple[str, s
...
 
-def store_metadata(uri: str, metadata: Dict[str, Union[str, dict, List[dict]]]) -> None:
-    # Store Transformation Engine metadata
-    api_client.set_property(uri, name="transform-version", value=metadata["int-tre-version"])
-    for key, version in extract_lambda_versions(metadata["lambda-functions-version"]):
-        # int-te-bagit-checksum-validation, int-te-files-checksum-validation, and int-text-parser-version
-        api_client.set_property(uri, name=f'transform-{key}', value=version)
+def store_metadata(uri: str, metadata: dict) -> None:
+    tdr_metadata = metadata["parameters"]["TDR"]
+
```
Comment (Floppy) [inline ds-caselaw-ingester/lambda_function.py:55]: I think we need to store `transform-version`, `int-te-bagit-checksum-validation`, `int-te-files-checksum-validation`, and `int-text-parser-version` still? They look like they've gone...

The checksum ones aren't particularly important, but I'd like to store `transform-version` (the version of the transofrmation engine itself) and `transform-parser-version` (the version of the parser inside it, `int-text-parser-version` in the payload)

File: ds-caselaw-ingester/lambda_function.py [hunk 2]
```diff
@@ -79,6 +89,8 @@ def send_retry_message(original_message: Dict[str, Union[str, int]], sqs_client:
             QueueUrl=os.getenv('SQS_QUEUE_URL'),
             MessageBody=json.dumps(retry_message)
         )
+    else:
+        print(f'Maximum number of retries reached for {original_message["consignment-reference"]}')
```
Comment (Floppy) [inline ds-caselaw-ingester/lambda_function.py:93]: Can we throw this up to Rollbar? Should be able to just raise an exception perhaps?

**Review by Floppy — CHANGES_REQUESTED** [review]

File: ds-caselaw-ingester/lambda_function.py (hunk 1 above)
Comment (LewisDaleUK) [reply ds-caselaw-ingester/lambda_function.py:55]: They're not part of the new metadata format it seems, which is why they've been removed

**Review by Floppy — CHANGES_REQUESTED** [review]
We should add the updated payload to https://github.com/nationalarchives/ds-caselaw-ingester/blob/main/aws_examples/s3/te-editorial-out-int/te-metadata.json too

File: ds-caselaw-ingester/lambda_function.py (hunk 1 above)
Comment (Floppy) [reply ds-caselaw-ingester/lambda_function.py:59]: > For now the new structure does not include them, but we are planning to include them in the next release

... ok then, we can add it back then, I guess.

**Review by Floppy — APPROVED** [review]

**Pull request merged**